dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -252,25 +248,129 @@ winrt::Control CommandBarFlyout::CreatePresenter()
+        {
+            presenterControl7.CornerRadius({ 4 });
+        }
+        else
+        {
+            presenterControl7.CornerRadius({ 0 });
+        }
     }
 
-    if (!SharedHelpers::Is21H1OrHigher())
+    // Since DropShadows don't play well with the entrance animation for the presenter,
+    // we'll need to fade it in. This name helps us locate the element to set the flag.
+    // We didn't want to apply this name to all FlyoutPresenters, which is why it's done
+    // in code behind here.
+    if (SharedHelpers::Is21H1OrHigher())
     {
-        // This logic applies to projected shadows, which are the default on < 21H1.
-        // When on 21H1 or higher, drop shadows are the default and need to be applied higher up
-        // in the tree to avoid being clipped away.  The default shadow works great for this.
-        // For < 21H1, we will provide our own shadow, not the one that FlyoutPresenter has by default.
-        // We need to specifically target the CommandBar for the shadow, not the default node far
-        // above that.
-        if (winrt::IFlyoutPresenter2 presenter2 = presenter)
+        presenter.Name(L"DropShadowFadeInTarget");
+    }
+
+    m_presenter.set(presenter);
+    m_commandBarOpeningRevoker = commandBar->Opening(winrt::auto_revoke, {
+    [this](auto const&, auto const&)
+    {
+        // When >21H1, if the CommandBar is opening, set the presenter's corner
+        // radius to 0 to let the CommandBar's corner radius show through.
+        if (SharedHelpers::Is21H1OrHigher())
         {
-            presenter2.IsDefaultShadowEnabled(false);
+            // During the SecondaryCommands's animations, drop shadows need to disappear.
+            // However, performing a Remove during Closing/Opening and a Add during Closed/Opened
+            // doesn't line up perfectly with the end of the animations. So, if there are animations
+            // we'll remove the shadow in the Closing and Opening events, but we'll put back the shadow
+            // inside of CommandBarFlyoutCommandBar's Storyboard Completed event handlers.
+            if (auto commandBar = winrt::get_self<CommandBarFlyoutCommandBar>(m_commandBar.get()))
+            {
+                if (commandBar->HasSecondaryOpenCloseAnimations())
+                {
+                    if (!AlwaysExpanded() && m_secondaryCommands.Size() > 0)
+                    {
+                        RemoveDropShadow();
+                    }
+                }
+            }
         }
     }
+    });
+
+    m_commandBarOpenedRevoker = commandBar->Opened(winrt::auto_revoke, {
+    [this](auto const&, auto const&)
+    {
+        if (winrt::IFlyoutBase5 thisAsFlyoutBase5 = *this)
+        {
+            // If we open the CommandBar, then we should no longer be in a transient show mode -
+            // we now know that the user wants to interact with us.
+            thisAsFlyoutBase5.ShowMode(winrt::FlyoutShowMode::Standard);
```

Comment:
> thisAsFlyoutBase5.ShowMode(winrt::FlyoutShowMode::Standard); [](start = 11, length = 61)

This seems unrelated to drop shadows.  Was this code moved?

Reply:
Yup, it was moved down a bit to be closer with the rest of the event handlers that I had to add.